### PR TITLE
Final adjustments

### LINKED ISSUE #23: Env Variables and Catch error
- [x] Use env variables (ImageInput.tsx, https://vitejs.dev/guide/env-and-mode.html) 

- [x] Carch error on Add Entry Page

### PR DESCRIPTION
**Pull request by riittagerhards**
- [x] Add useMemo on Gallery page
- [x] Add scroll element "to top" on Gallery page
- [x] Add animation when moving from page to page 💫
- [x] Catch error on Add entry page
- [x] Fix a bug in Button component

fixes #23

### CODE REVIEW

File: src/app/pages/Gallery/ToTopIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+function ToTopIcon({ className, onClick }: ToTopIconProps): JSX.Element {
+  return (
+    <svg
+      className={className}
+      onClick={onClick}
+      width="24"
+      height="30"
+      viewBox="0 0 24 30"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
```
Comment (AnkePfei2fer) [inline src/app/pages/Gallery/ToTopIcon.tsx:15]:
```suggestion
      
```

File: src/app/pages/Gallery/ToTopIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+function ToTopIcon({ className, onClick }: ToTopIconProps): JSX.Element {
+  return (
+    <svg
+      className={className}
+      onClick={onClick}
+      width="24"
+      height="30"
+      viewBox="0 0 24 30"
+      fill="none"
```
Comment (AnkePfei2fer) [inline src/app/pages/Gallery/ToTopIcon.tsx:14]: I'm not sure if you really need this line, as you specify fill in the path below.

File: src/app/pages/SingleEntry/EmptyIcon.tsx [hunk 3]
```diff
@@ -1,28 +1,89 @@
...
   return (
     <svg
-      width="96"
-      height="109"
-      viewBox="0 0 96 109"
+      width="103"
+      height="98"
+      viewBox="0 0 103 98"
       fill="none"
       xmlns="http://www.w3.org/2000/svg"
```
Comment (AnkePfei2fer) [inline src/app/pages/SingleEntry/EmptyIcon.tsx:8]:
```suggestion
      
```

File: src/app/pages/SingleEntry/SingleEntry.tsx [hunk 4]
```diff
@@ -42,6 +43,7 @@ function SingleEntry(): JSX.Element {
   const deleteEntry = useDeleteEntry(deleteDate);
 
   async function handleClick() {
+    console.log('testing');
```
Comment (AnkePfei2fer) [inline src/app/pages/SingleEntry/SingleEntry.tsx:46]: You probably can remove it

**Review by AnkePfei2fer — COMMENTED** [review]
LGTM 🚀 
To make your code less crowded, you could think of making components out of your svg.
Here's how I did it:
https://github.com/AnkePfei2fer/steinchen/commit/6ee67ceea8cb24a996eebaf31a336a92019f6a88

File: src/app/pages/Gallery/ToTopIcon.tsx (hunk 2 above)
Comment (riittagerhards) [reply src/app/pages/Gallery/ToTopIcon.tsx]: 👍

File: src/app/pages/SingleEntry/EmptyIcon.tsx (hunk 3 above)
Comment (riittagerhards) [reply src/app/pages/SingleEntry/EmptyIcon.tsx:8]: 👍

File: src/app/pages/SingleEntry/SingleEntry.tsx (hunk 4 above)
Comment (riittagerhards) [reply src/app/pages/SingleEntry/SingleEntry.tsx]: 👍

**Pull request merged**